packages/core/src/domain/session/sessionManager.ts

```diff
@@ -15,6 +15,8 @@ export interface SessionManager<TrackingType extends string> {
     startTime?: RelativeTime,
     predicate?: (entry: ValueHistoryEntry<SessionContext<TrackingType>>) => boolean
   ) => SessionContext<TrackingType> | undefined
+  renewObservable: Observable<string>
+  expireObservable: Observable<string>
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** use `TrackingType` instead of `string`
‏